src/_includes/layouts/page.njk

```diff
@@ -7,4 +7,22 @@
         <a>Lists</a>
         <a>About</a>
     </nav>
+{% endblock %}
+
+
+{% block footer %}
+    <nav class="full-width">
+        <div class="breakout">
+            <div> 
+                <a href="/">Home</a>

```

Comment:
i swapped out "about" for "projects" will need to reflect this change in the header as well.